central/vulnmgmt/service/service_impl_benchmark_test.go

```diff
@@ -0,0 +1,94 @@
+//go:build sql_integration
+
+package service
+
+import (
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stackrox/rox/central/testutils"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/generated/storage"
+	"google.golang.org/grpc"
+)
+
+func BenchmarkService_Export(b *testing.B) {
+	testHelper := &testutils.ExportServicePostgresTestHelper{}
+	err := testHelper.SetupTest(b)
+	if err != nil {
+		b.Error(err)
+	}
+	defer testHelper.TearDownTest(b)
+	svc := New(testHelper.Deployments, testHelper.Images)
+	deltas := []int{500}
+	// The test runs by default with a lower scale as smoke test
+	// in the benchmark unit tests. To test at higher scales (takes time),
+	// run the test with ROX_SCALE_TEST set to a non-empty value
+	// in the test environment.
+	scale := os.Getenv("ROX_SCALE_TEST")
+	if scale != "" {
+		deltas = []int{500, 500, 1000, 3000, 5000}
```

Comment:
Why is `500` run twice?

Reply:
These are deltas for the overall dataset generation.
The benchmarks are run on the same database, populated incrementally.
The resulting dataset sizes are 500, 1000, 2000, 5000, 10000.

Reply:
Ah I see. Maybe it would be easier to represent the test cases in totals instead of deltas to avoid this confusion?